test/integration/patients/sidebar/action-sidebar.js

```diff
@@ -721,9 +721,34 @@ context('action sidebar', function() {
       },
     });
 
+    const testFiles = [
```

Comment:
I think it's maybe worth going ahead and making the support/api/files with getFile and testFile instead of needing the uuid everywhere.  Maybe a subsequent PR

Reply:
For sure, I created a story for that: https://app.shortcut.com/roundingwell/story/61236.